server/neptune/temporalworker/config/config.go

```diff
@@ -30,10 +30,11 @@ type TerraformConfig struct {
 
 // Config is TemporalWorker specific user config
 type Config struct {
-	AuthCfg      AuthConfig
-	ServerCfg    ServerConfig
-	TemporalCfg  valid.Temporal
-	TerraformCfg TerraformConfig
+	AuthCfg            AuthConfig
+	ServerCfg          ServerConfig
+	TemporalCfg        valid.Temporal
+	TerraformCfg       TerraformConfig
+	LogStreamingJobCfg valid.Jobs
```

Comment:
```suggestion
	JobCfg valid.Jobs
```

No need to mention Log streaming everywhere, that's an implementation detail.